HedgeModManager/UI/MainWindow.xaml.cs

```diff
@@ -1886,54 +1983,174 @@ private void CodesList_GridViewColumnHeader_Click(object sender, RoutedEventArgs
 
         private void UpdateCodeDescription(Code code)
         {
-            var theme = new ResourceDictionary();
+            var theme = new ResourceDictionary()

```

Comment:
Is there any reason for opening a resource dictionary this way? HedgeApp.Current.FindResource() should do the trick for getting the required resources.

Reply:
No reason, will replace now.